examples/social-share.amp.html

```diff
@@ -43,6 +36,8 @@
 
     <div class="social-box">
       <amp-social-share type="email" data-param-subject="Hello World" data-param-body="What's up?"></amp-social-share>
+      <amp-social-share type="whatsapp"></amp-social-share>
+      <amp-social-share type="pinterest"></amp-social-share>
```

Comment:
Done. Also filed an issue for https://github.com/ampproject/amp-by-example/issues/542